config/locales/sufia.en.yml

```diff
@@ -2,7 +2,7 @@ en:
   curation_concerns:
     institution_name:   "CHF"
   sufia:
-    product_name:       "CHF Digital Collections (Beta)"
+    product_name:       "Digital Collections"
```

Comment:
Same, if above in i18n is changed, this should probably match, I don't really know where one is used vs the other, matching seems a good bet.